ports/botan/portfile.cmake

```
@@ -177,7 +177,7 @@ else()
     endif()
 endif()
 
-vcpkg_cmake_config_fixup(CONFIG_PATH lib/cmake/Botan-3.3.0)
+vcpkg_cmake_config_fixup(CONFIG_PATH lib/cmake/Botan-3.5.0)
```

```suggestion
vcpkg_cmake_config_fixup(CONFIG_PATH "lib/cmake/Botan-${VERSION}")
```
The commit message says pkgconfig but this isn't pkgconfig.